AirCasting/SessionViews/SessionCartView.swift

```diff
@@ -101,14 +101,16 @@ private extension SessionCartView {
             isMapButtonActive = true
```

Comment:
this shouldn't be `isMapButtonActive` but `isGraphButtonActive`. 
It's rebase's fault. I hate rebases.

Reply:
Feel You, Hate rebases too. Fixing now!

Reply:
<grumpy 👴🏻 Paweł>
The conflicts emerging in each PR are because we're shoving everything in the same files (mainly views are our god objects, but not only). We need to split responsibilities and make our architecture obey clean code principles sooner than later if we don't want to push our lives into a misery ;)
</grumpy 👴🏻 Paweł>

Reply:
I love your comments and I know it is serious one. I think that as soon as after MVP release we can focus on that mostly 😄

Reply:
Could we make a deal that after we're done with MVP (essential) we'll spend a day on a call refactoring code structure? I'd love to learn more about code structure from someone experienced :D